src/xtgeo/grid3d/_grid_etc1.py

```diff
@@ -221,16 +231,21 @@ def get_bulk_volume(self, name="bulkvol", asmasked=True, precision=2):
     )
 
     if asmasked:
-        bval = np.ma.masked_greater(bval, xtgeo.UNDEF_LIMIT)
+        bval = np.ma.masked_greater(bval, UNDEF_LIMIT)
 
-    bulk.values = bval
+    bulk.values = bval  # type: ignore
```

Comment:
Why the ignore?

Reply:
This is a tricky one. The values setter have a different type (np.ndarray) then the values property, which is np.ma.MaskedArray.  And mypy complains... 
Inside our setter function we have a validation and conversion of the input array to an MaskedArray.
Doesn't seem that this is going to be supported by mypy:
https://github.com/python/mypy/issues/3004